core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -182,7 +182,7 @@ describe('Enable correctness feedback and set correctness', function() {
       // Go back to main tab to create interactions.
       await explorationEditorPage.navigateToMainTab();
       await explorationEditorMainTab.setStateName('First');
-      await explorationEditorMainTab.setContent(await forms.toRichText(
+      await explorationEditorMainTab.setContent1(await forms.toRichText(
```

Comment:
Used `setContent1` instead of `setContent` because `protractor-test-fade-in` is present only in the exploration editor page and therefore used a different `setContent1` for the exploration editor page to handle the flake which happens in the exploration editor page only.

Moreover there are many occurences of `setContent` in `coreEditorAndPlayerFeatures` but used `setContent1` in just two places as stateContent editor flake occured in these places only.